sympy/functions/elementary/complexes.py

```diff
@@ -1000,6 +1206,15 @@ class principal_branch(Function):
     >>> principal_branch(exp_polar(2*pi*I)*3*z, 2*pi)
     3*principal_branch(z, 2*pi)
 
+    Parameters
+    ==========
+
+    x : Expr
+        A polar number.
+
+    period : Expr
+        Positive real number of infinity.
```

Comment:
```suggestion
        Positive real number or infinity.
```

Reply:
Changed.

Reply:
If this was a typo, then the above summary should be changed too

Reply:
Done.